src/schemas/backend/UpdateTimesheet.ts

```diff
@@ -0,0 +1,73 @@
+import { z } from "zod";
+import { RowSchema, CommentSchema, ScheduledRowSchema } from "../RowSchema"
+import * as dbtypes from './Timesheet'
+
+/*
+-------------------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+---------------------------------DELETE WHEN MONOREPO CREATED------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------
+
+*/
+
+// Currently supported timesheet operations 
+export const enum TimesheetOperations {
+    INSERT = "INSERT", 
+    UPDATE = "UPDATE", 
+    DELETE = "DELETE", 
+    STATUS_CHANGE = "STATUS_CHANGE", 
+    CREATE_TIMESHEET = "CREATE_TIMESHEET"
+} 
+
+
+
+export const enum TimesheetListItems {
```

Comment:
Could you add in some quick comments for the enums here? I think they're hopefully self-explanatory, but will be nice to have it documented for new devs. Maybe also just give a couple sentence summary at the top of this file explaining what the whole idea behind having these CRUD-like operations is?

Reply:
Yeah makes sense will update it in the backend / leave comment to check there instead as its shared code / will be pointing to backend at some point with monorepo.